examples/images_contours_and_fields/image_demo.py

```diff
@@ -121,26 +122,21 @@
 # suggested.
 
 A = np.random.rand(5, 5)
-plt.figure(1)
-plt.imshow(A, interpolation='nearest')
-plt.grid(True)
-
-plt.figure(2)
-plt.imshow(A, interpolation='bilinear')
-plt.grid(True)
 
-plt.figure(3)
-plt.imshow(A, interpolation='bicubic')
-plt.grid(True)
+fig, axs = plt.subplots(1, 3, figsize=(10, 3))
+for ax, interp in zip(axs, ['nearest', 'bilinear', 'bicubic']):
+    ax.imshow(A, interpolation=interp)
+    ax.set_title(interp.capitalize())
+    ax.grid(True)
 
 plt.show()
 
 
 ###############################################################################
 # You can specify whether images should be plotted with the array origin
-# x[0,0] in the upper left or upper right by using the origin parameter.
+# x[0,0] in the upper left or lower right by using the origin parameter.
 # You can also control the default be setting image.origin in your
```

Comment:
I'm gonna mark typeos w/ `TYPEO` in the comment as a reminder to go back @ the end and fix the typos just before merging. (there's one in this line) "control the default be setting"